modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of the Jaeger Operator.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+EOF
+----
+
+. Get the `jaeger-collector` service endpoint and take note of it for use in the next step:
++
+----
+$ oc get svc
+----
+
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
++
+.Tracing YAML example
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    tracing:
+      sample-rate: "0.1"
+      backend: zipkin
+      zipkin-endpoint: http://jaeger-collector.<JAEGER_NAMESPACE>.svc.cluster.local:9411/api/v2/spans
+      debug: "true"
+----
++
+.. The backend must be `zipkin`.
+.. The `zipkin-endpoint` must point to your `jaeger-collector` service endpoint from the previous step.
+
+.Verification  steps
+Access the Jaeger web console to see tracing data. You can access the Jaeger web console by using the `jaeger-collector` service endpoint.
```

Comment:
There's an issue tracking docs for accessing the Jaeger console https://issues.redhat.com/browse/OSSMDOC-91 - so maybe I can leave this here for now and then switch to including that doc module once it exists? Should this be the same thing @nak3 ?